### ISSUE TITLE
Performance Profile Automation

### ISSUE THREAD

**Issue by Terrigible** [franciscofranco/FK-Kernel-Manager-IssuesAndFeatureRequest#57]
Please bring back performance profile automation.
Asked you once but you didn't add it back. Afraid you forgot about it.
Thanks.

**Comment by franciscofranco** [comment]
Yes, it's coming, I'm working on more automation integration for a bigger update.
